### DISCUSSION TITLE
Email Provider magic links

### DISCUSSION THREAD

**Discussion by gleysonabreu** [nextauthjs/next-auth#10074] [Help]
Can I use magic links login with an existing backend without needing an adapter like Prisma for example?

```
import EmailProvider from "next-auth/providers/email";
...
providers: [
  EmailProvider({
    server: {
      host: process.env.EMAIL_SERVER_HOST,
      port: process.env.EMAIL_SERVER_PORT,
      auth: {
        user: process.env.EMAIL_SERVER_USER,
        pass: process.env.EMAIL_SERVER_PASSWORD
      }
    },
    from: process.env.EMAIL_FROM
  }),
]
```

For example, using Credentials we have the authorize() function to connect with the external backend, in EmailProvider do we have any option for this? Thanks.

**Comment by chinnuu05** [comment]
Also looking for an answer to this, made a similar post asking how to use external Django API backend with EmailProvider
